docs/vanilla.md

```diff
@@ -35,32 +35,138 @@ of VDDK Programming Guide. **Note: please apply privileges at the vCenter Server
 
 1. [Install Velero](https://velero.io/docs/v1.5/basic-install/)
 2. [Install Object Storage Plugin](#install-object-storage-plugin)
-3. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
+3. [Create VC Credential Secret](#create-vc-credential-secret)
+4. [Create Velero vSphere Plugin Config](#create-velero-vsphere-plugin-config)
+5. [Install Velero Plugin for vSphere](#install-velero-plugin-for-vsphere)
 
 ### Install Object Storage Plugin
 
 Volume backups are stored in an object store bucket. They are stored in the same bucket configured for the object storage plugin of Velero. Before installing the vSphere plugin, a Velero object storage plugin is required.
 
 Currently, only AWS plugin is supported and compatible with vSphere plugin. Please refer to [velero-plugin-for-aws](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/master/README.md) for more details about using **AWS S3** as the object store for backups. S3-compatible object stores, e.g, **MinIO**, are also supported via AWS plugin. Please refer to [install with MinIO](https://velero.io/docs/v1.5/contributions/minio/).
 
+### Create VC Credential Secret
+
+You can retrieve these values from Secret `vsphere-config-secret` in `vmware-system-csi` namespace for
+2.3 vSphere CSI driver OR Secret `vsphere-config-secret` in `kube-system` namespace for 2.2 vSphere CSI
+driver
```

Comment:
What if vSphere CSI Driver is deployed as a process?

Reply:
Added a line stating that they need to contact vsphere admin to retrieve the info in other cases.